File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/utils/PGPUtil.java
package tw.nekomimi.nekogram.utils;
import android.content.Intent;
import java.io.InputStream;
import java.io.OutputStream;
public class PGPUtil {
    public static PGPApi api = new PGPApi();
    public static void post(Runnable r) { if (r != null) r.run(); }
    public static class PGPApi {
        public void executeApiAsync(Intent intent, InputStream input, OutputStream output, Object callback) {}
    }
}

File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/utils/PGPUtil.kt
package tw.nekomimi.nekogram.utils

import org.openintents.openpgp.util.OpenPgpApi
import org.openintents.openpgp.util.OpenPgpServiceConnection

object PGPUtil {

    lateinit var serviceConnection: OpenPgpServiceConnection
    lateinit var api: OpenPgpApi

    @JvmStatic
    fun recreateConnection() {}

    @JvmStatic
    fun post(runnable: Runnable) {}
}
